tests/browser/affiliations/change_secondary_as_primary_affiliation.robot

```diff
@@ -14,15 +14,16 @@ Create Secondary Affiliation for Contact
     Page Should Contain    &{contact}[LastName]
     Page Should Contain    &{account}[Name]
     Scroll Page To Location    50    600
-    Sleep    5
+    #Sleep    5
     Click Id    &{account}[Name]
-    Sleep    5
+    #Sleep    5
     Scroll Page To Location    0    400
     Click Edit Button    Edit Primary
-    Sleep    5
+    #Sleep    5
     Select Checkbox    //div[contains(@class, "uiInput")]/label/following-sibling::input[@type='checkbox']
     Click Record Button    Save
     Sleep    5
```

Comment:
Not sure if you wanted to comment this one out too, or if this sleep is needed.

Reply:
I don't think the one on line 25 is needed, but I think we do need something to wait on line 22.

The difference is that clicking the save button triggers an XMLHTTPRequest, so the built-in wait-for-requests-to-complete-after-click logic should take care of waiting for that one. On the other hand, clicking on the edit button does not trigger a request, but does take some time to render the edit version of the checkbox. It would be better to have a line that waits for the checkbox locator to appear instead of an arbitrary sleep though.

Reply:
The one on Line 25 is included on purpose. Once I hit save the change is not reflecting immediately when I refresh page to verify under details tab. I am having to wait for 5 seconds before I verify. I have run the test case multiple times and I do not see any issue with removing the sleep after clicking edit button. @davisagli if you think it would be good to add delay there I have a keyword to wait for locator and I can include that in the code.

Reply:
@skristem okay we can leave it as is for now. I have some more testing to do and will let you know if I still have trouble with it.